sky/tools/create_macos_framework.py

```diff
@@ -152,26 +164,29 @@ def process_framework(dst, args, fat_framework, fat_framework_binary):
   # Zip FlutterMacOS.framework.
   if args.zip:
     filepath_with_entitlements = ''
-    filepath_without_entitlements = 'FlutterMacOS.framework/Versions/A/FlutterMacOS'
+
+    framework_dst = os.path.join(dst, 'FlutterMacOS.framework')
+    filepath_without_entitlements = 'Versions/A/FlutterMacOS'
```

Comment:
The metadata path will now be `FlutterMacOS.framework.zip/Versions/A/FlutterMacOS`
so L169 will be like `filepath_without_entitlements = FlutterMacOS.framework.zip/Versions/A/FlutterMacOS`

Reply:
ack

Reply:
Actually adding this seems overcomplicated. As a workaround I'll check the file name explicitly in the recipes with a TODO to remove it after the outer zip is deleted.